modules/toolbox/__init__.py

```diff
@@ -23,7 +24,7 @@
 from sklearn.tree import DecisionTreeRegressor
 from sklearn.ensemble import AdaBoostRegressor
 
-__all__ = ["framework_tools", "setup", "csv", "pd", "np",
+__all__ = ["data_io", "data_splitting", "setup", "csv", "pd", "np",
```

Comment:
Framework tools was an old mess of functions (utils), I split it up into data saving/loading, and data splitting (like testing/training sets).